src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -337,6 +338,15 @@ private static boolean hasLineBreakBefore(DetailAST rightCurly) {
      */
     private static final class Details {
 
+        /**
+         * Token types that identify definition tokens.
+         */
+        private static final int[] DEF_TOKENS = {
```

Comment:
Please convert this into a Set or an explicitly sorted array.

Reply:
Done